docs/src/checkout.md

```diff
@@ -8,7 +8,7 @@ curl --request POST \
   --header 'cache-control: no-cache' \
   --header 'postman-token: e0dee5b6-94b0-0dd3-f67d-b88804ec5b74' \
   --header 'token: 123' \
-  --data '{"contactEmail" : "mschwartz@test.com","addressToken" : "adrs_xipKF02Td5eJ66IoEyzHdXUM","contactFirstName":"Moises","contactLastName","Scheiyz"}'
+  --data '{"contactEmail" : "123456@test.com","addressToken" : "adrs_xipKF02Td5eJ66IoEyzHdXUM","contactFirstName":"####","contactLastName","####"}'

```

Comment:
'###' are only used to represent digits, not text. So, in place of a name just use a typical name eg John Doe, or John Smith etc.